cmd/cluster-agent/admission/server.go

```diff
@@ -40,14 +43,24 @@ const jsonContentType = "application/json"
 
 // Request contains the information of an admission request
 type Request struct {
-	// Raw is the raw request object
-	Raw []byte
+	// UID is the unique identifier of the AdmissionRequest
+	UID types.UID
 	// Name is the name of the object
 	Name string
 	// Namespace is the namespace of the object
 	Namespace string
+	// Kind is the kind of the object
+	Kind metav1.GroupVersionKind
+	// Resource is the resource of the object
+	Resource metav1.GroupVersionResource
```

Comment:
I don't see where this is used. Maybe I missed it.

Reply:
Done: https://github.com/DataDog/datadog-agent/pull/30715/commits/d3bf9f04a4ff1b11b5c1b17746b317501418cb92